src/components/ProductColourButton.tsx

```diff
@@ -12,6 +8,11 @@ type ProductColourButtonProps = {
   setCardImage: React.Dispatch<React.SetStateAction<string>>;
 };
 
+type Content = {
+  colours: string[];
+  image: string[];
+};
+
```

Comment:
Doesn't this conflict with the naming from the import?

import { Content } from "./ProductCard";

Reply:
removed type. Only importing Content type from ProductCard